README.md

```diff
@@ -1,11 +1,15 @@
 # Java Tree-sitter
 
-[![CI][ci]](https://github.com/tree-sitter/java-tree-sitter/actions/workflows/ci.yml)
-[![central][central]](https://central.sonatype.com/artifact/io.github.tree-sitter/jtreesitter)
-[![docs][docs]](https://tree-sitter.github.io/java-tree-sitter/)
+[![CI][ci-shield]](https://github.com/tree-sitter/java-tree-sitter/actions/workflows/ci.yml)
+[![central][central-shield]](https://central.sonatype.com/artifact/io.github.tree-sitter/jtreesitter)
+[![docs][docs-shield]][docs]
 
 Java bindings to the [tree-sitter] parsing library.
 
+## Usage
```

Comment:
This seems superfluous. Just click the badge.

Reply:
My idea here was to make it a bit easier for users to discover.

Otherwise new users might not know how to get started with this project, and might not consider looking at the Javadoc (in case they do notice the badge). They might not expect such an extensive usage example in the package Javadoc because not all projects have that (respectively have it in separate documentation or in the README).